include/osm_store.h

```diff
@@ -296,7 +309,7 @@ class OSMStore
 				return;
 			} catch(boost::interprocess::bad_alloc &e) {
 				map_size = map_size * 2;
-				std::cout << "Resizing osm store to size: " << map_size << std::endl;
+				std::cout << "Resizing osm store to size: " << (map_size / 1000000) << "M                " << std::endl;
```

Comment:
The actual filesize of the osm_store.dat file appears to be twice the reported value - maybe change 1000000 to 500000?

Reply:
Yes, i noticed this too. I am not sure why this is, I do give this value to boost interprocess grow method. If interprocess allocates 2 * size I specify, i would rather ask for size / 2 there. Because otherwise it would be confusing.

Reply:
https://github.com/systemed/tilemaker/blob/master/include/osm_store.h#L297-L307

Reply:
Oh, wait. It grows with _extra_ bytes, not to the specified size

````C++
   //!Tries to resize mapped file so that we have room for
   //!more objects.
   //!
   //!This function is not synchronized so no other thread or process should
   //!be reading or writing the file
   static bool grow(const char *filename, size_type extra_bytes)
   {
      return base_t::template grow
         <basic_managed_mapped_file>(filename, extra_bytes);
   }